pysnooper/tracer.py

```diff
@@ -185,6 +188,14 @@ class Tracer:
 
         @pysnooper.snoop(custom_repr=((type1, custom_repr_func1), (condition2, custom_repr_func2), ...))
 
+    Customize whether truncate the variable value or not::
+
+        @pysnooper.snoop(truncate=False)
```

Comment:
This doesn't match the `need_truncate` parameter.

Reply:
Good catch.